src/System.Windows.Forms/src/System/Windows/Forms/TaskDialogButtonCollection.cs

```diff
@@ -46,8 +46,22 @@ public TaskDialogButton Add(string? text, bool enabled = true, bool allowCloseDi
         }
 
         /// <inheritdoc/>
+        /// <exception cref="ArgumentNullException">
+        ///   <paramref name="item"/> is <see langword="null"/>
+        /// </exception>
+        /// <exception cref="ArgumentException">
+        ///   <paramref name="item"/> has already been added to the collection.
+        /// </exception>
+        /// <exception cref="InvalidOperationException">
+        ///   <paramref name="item"/> is already a part of a different collection.
+        /// </exception>
```

Comment:
@BillWagner @gewarren will this work, or do we need to import the original docs?
Here we're subclassing https://docs.microsoft.com/en-us/dotnet/api/system.collections.objectmodel.collection-1.add?view=netcore-3.1

Reply:
inheritdoc still doesn’t work on mdoc. You could also keep here as-is and copy the docs on the API repo.

Reply:
Thank you.

@kpreisser we should be good to go.